src/sas/qtgui/Utilities/TabbedModelEditor.py

```diff
@@ -205,7 +205,7 @@ def loadFile(self, filename):
         self.editor_widget.txtEditor.setToolTip("")
 
         # See if there is filename.c present
-        c_path = Path(str(self.filename.parent) + self.filename.name.replace(".py", ".c"))
+        c_path = Path(str(self.filename.parent)) / self.filename.name.replace(".py", ".c")
```

Comment:
Even better. `self.filename.parent` is already a Path object. This means we can just do

```
c_path = self.filename.parent / self.filename.name.replace(".py", ".c")
```